CONTRIBUTING.md

```diff
@@ -2,8 +2,10 @@
 
 ## Contribution workflow
 
-Hubble uses GitHub for collaborative development. Please use GitHub issues to
-discuss proposals and use pull requests to suggest changes.
```

Comment:
Did you mean to remove this line?

> Please use GitHub issues to discuss proposals and use pull requests to suggest changes.

Reply:
I initially removed this because I considered it covered in the Development Guide, but I think you're right: the sentence should stay. I've re-added it.